TA_Scheduler_App/test_skills_feature.py

```diff
@@ -0,0 +1,36 @@
+#import unittest
+from django.test import TestCase
+from TA_Scheduler_App.models import User, skills
+from TA_Scheduler_App.skills_features import SkillsFeatures
+
+
+
+class MyTestCase(TestCase):
```

Comment:
The tester found all three tests, but the test result summary did not display which tests passed, just how many. I suspect it is because this className was not changed to follow Django test conventions. 
```suggestion
class Test_Skills_Feature(TestCase):
```

Reply:
This is untrue, running tests by default will only show failed tests.  To change this, you can turn on the "Show Passed" toggle in the Run panel.  The toggle is visualized as a check mark.